hash/saturn.xml

```diff
@@ -30330,7 +30330,7 @@ Olympic Soccer (Fra) T-7904H-09
 	<!-- Identifying 0691-Atlantis-fre-cd1... -->
 	<!-- Identifying 0691-Atlantis-fre-cd2... -->
 	<software name="atlantisf" cloneof="atlantis" supported="no">
-		<description>Atlantis - Secrets d'un monde oublié (Fra)</description>
+		<description>Atlantis - Secrets d'un Monde Oublié (Fra)</description>
```

Comment:
The cover art for this uses a colon to separate the title and subtitle, with a space before the colon, and only capitalises the first letter of the subtitle: https://www.youtube.com/watch?v=EgalDwWbhCI

Reply:
I removed the capitalized letters, keeping only the first subtitle's word.
I kept the dash between the title and subtitle
https://www.amazon.fr/Atlantis-Secret-monde-oubli%C3%A9-Saturn/dp/B01N4J84RK

Reply:
What are you using from that to justify the use of a dash?  It’s not using punctuation at all there, relying on font size to make the distinction.  In cases like that MAME typically uses a colon anyway.  For example in model3.cpp, the Virtua On 2 games use a colon to separate the title and subtitle even though media just displays the subtitle in a different, smaller font.

Reply:
I used the dash for no particular reason, I just putted like it was before I changed it at the first place. But it's fine to me to use the colon character. So I changed it again.
But in this case, the parent set "Atlantis - The Lost Tales (Euro)" should have the same treatment, am I correct?

Reply:
Well, it does seem to be conventionally rendered that way, e.g. here: https://en.wikipedia.org/wiki/Atlantis:_The_Lost_Tales

Also, the French title should use French rules for spacing around punctuation, not English rules.  You place a space before a colon (les deux-points) in French, but not in English:
* See the title of the page here: https://fr.wikipedia.org/wiki/Atlantis_:_Secrets_d%27un_monde_oubli%C3%A9
* See French usage examples here: https://study.com/academy/lesson/french-punctuation-marks-rules.html